data/assets/examples/screenspacerenderable/screenspacetimevaryingimageonline/data/dummydata.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Maybe rename the file to "example.json" or something?  Since it's not really 'dummy' per se